tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -394,7 +505,11 @@ def test_read_csv_with_special_chars_in_format_type_options(session, mode):
     # without the setting it should fail schema validation
     df2 = get_reader(session, mode).schema(schema1).csv(test_file)
     with pytest.raises(SnowparkSQLException) as ex_info:
-        df2.collect()
+        try:
+            df2.collect()
+        except Exception as e:
```

Comment:
Why add a print?

Reply:
ah.. this should be removed, I added it for debugging